index.js

```diff
@@ -299,10 +299,19 @@ const extractFeatures = async page => {
   }
 }
 
+let sharp = null
+const resizeCanvas = async (canvas, resX, resY) => {
```

Comment:
If we're being picky `canvas` can be a bit misleading here, as we are passing an image-as-buffer, not a canvas element. Would clarify using `image` instead.